qemu/tests/block_copy.py

```diff
@@ -321,3 +321,48 @@ def clean(self):
         while self.trash_files:
             tmp_file = self.trash_files.pop()
             utils.system("rm -f %s" % tmp_file, ignore_status=True)
+
+    def wait_for_complete(self):
+        """
+        Check block job status, utils timeout; if still not complete,
+        raise TestFail exception.
+        """
+        params = self.parser_test_args()
+        timeout = params.get("wait_timeout")
+        if self.vm.monitor.protocol == "qmp":
+            self.vm.monitor.clear_event("BLOCK_JOB_COMPLETED")
+        complete = utils_misc.wait_for(lambda: self.is_complete(),
+                                       first=3.0, step=3.0, timeout=timeout)
+        if not complete:
+            raise error.TestFail("Wait block job ready "
+                                 "timeout in %ss" % timeout)
+
+    def is_complete(self):
+        """
+        Check block job completed or not.
+        :return block job status.
+        """
+        return self.vm.monitor.get_event("BLOCK_JOB_COMPLETED")
+
+    def create_file(self):
+        """
+        Create files and record m5 values of them.
+        """
+        params = self.params
+        session = self.get_session()
+        file_names = params["file_names"]
+        file_create_cmd = params.get("create_cmd", "touch %s")
+        test_exists_cmd = params.get("test_exists_cmd", "test -f %s")
+        for file in file_names.split():
+            if session.cmd_status(test_exists_cmd % file):
+                session.cmd(file_create_cmd % file)
+        session.cmd("md5sum %s > record.md5" % (file_names))
```

Comment:
Thanks, but as I explained above, "md5sum test1 test2" will sum both test1 and test2 to record.md5. I think in this way I can save a 'for' loop, even it is hard to specify the name as %s.md5, since file_names is like "/mnt/sda/test /mnt/sdb/test", but I think it is more efficiency to use record.md5sum instead of add a 'for' loop.